backend/flow_api/flow/shared/hook_oauth_links.go

```diff
@@ -0,0 +1,63 @@
+package shared
+
+import (
+	"fmt"
+	"github.com/labstack/echo/v4"
+	"github.com/teamhanko/hanko/backend/flowpilot"
+	"net/url"
+)
+
+type OAuthLinks struct {
```

Comment:
I would like to suggest adding a verb to the hook action name, e.g `GenerateOAuthLinks` (instead of `OAuthLinks`)